include/deal.II/dofs/dof_accessor.h

```diff
@@ -2019,6 +2019,15 @@ class DoFCellAccessor : public DoFAccessor<DoFHandlerType::dimension,
   void
   clear_p_coarsen_flag() const;
 
+  /**
+   * Returns the active_fe_index that this cell would have if refinement would
+   * have happened. Prediction is based on the current p-refinement flags set on
+   * this cell.
+   *
+   * @note The DoFHandler in use has to be of type hp::DoFHandler.
+   */
+  unsigned int
+  active_fe_index_after_refinement() const;
```

Comment:
`after_p_refinement()`

Reply:
Long, but inevitable for a proper description I guess.

Reply:
> Long

luckily we don't use Fortran in deal.II 😺